openquake/qa_tests_data/classical_risk/case_4/expected/loss_curves-000.csv

```diff
@@ -1,158 +1,158 @@
-a1,structural,8.01244E+07,4.86169E-09
-a1,structural,8.04879E+07,4.66265E-09
-a1,structural,8.08515E+07,4.47219E-09
-a1,structural,8.12150E+07,4.28991E-09
-a1,structural,8.15785E+07,4.11543E-09
-a1,structural,8.18872E+07,3.97313E-09
-a1,structural,8.21960E+07,3.83599E-09
-a1,structural,8.25047E+07,3.70383E-09
-a1,structural,8.28134E+07,3.57644E-09
-a1,structural,8.31222E+07,3.45366E-09
-a1,structural,8.33849E+07,3.35266E-09
-a1,structural,8.36476E+07,3.25476E-09
-a1,structural,8.39103E+07,3.15985E-09
-a1,structural,8.41730E+07,3.06785E-09
-a1,structural,8.44358E+07,2.97866E-09
-a1,structural,8.46598E+07,2.90474E-09
-a1,structural,8.48839E+07,2.83275E-09
-a1,structural,8.51079E+07,2.76262E-09
-a1,structural,8.53320E+07,2.69432E-09
-a1,structural,8.55560E+07,2.62778E-09
-a1,structural,8.57476E+07,2.57227E-09
-a1,structural,8.59391E+07,2.51799E-09
-a1,structural,8.61306E+07,2.46490E-09
-a1,structural,8.63221E+07,2.41299E-09
-a1,structural,8.65136E+07,2.36222E-09
-a1,structural,8.66777E+07,2.31962E-09
-a1,structural,8.68418E+07,2.27782E-09
-a1,structural,8.70059E+07,2.23680E-09
-a1,structural,8.71699E+07,2.19656E-09
-a1,structural,8.73340E+07,2.15707E-09
-a1,structural,8.74749E+07,2.12376E-09
-a1,structural,8.76158E+07,2.09098E-09
-a1,structural,8.77567E+07,2.05874E-09
-a1,structural,8.78976E+07,2.02701E-09
-a1,structural,8.80385E+07,1.99579E-09
-a1,structural,8.81597E+07,1.96932E-09
-a1,structural,8.82810E+07,1.94322E-09
-a1,structural,8.84023E+07,1.91749E-09
-a1,structural,8.85235E+07,1.89210E-09
-a1,structural,8.86448E+07,1.86707E-09
-a1,structural,8.87494E+07,1.84576E-09
-a1,structural,8.88540E+07,1.82469E-09
-a1,structural,8.89586E+07,1.80388E-09
-a1,structural,8.90632E+07,1.78332E-09
-a1,structural,8.91678E+07,1.76300E-09
-a1,structural,8.92583E+07,1.74563E-09
-a1,structural,8.93487E+07,1.72844E-09
-a1,structural,8.94391E+07,1.71142E-09
-a1,structural,8.95296E+07,1.69458E-09
-a1,structural,8.96200E+07,1.67791E-09
-a1,structural,8.96984E+07,1.66360E-09
-a1,structural,8.97768E+07,1.64942E-09
-a1,structural,8.98551E+07,1.63537E-09
-a1,structural,8.99335E+07,1.62144E-09
-a1,structural,9.00118E+07,1.60764E-09
-a1,structural,9.00799E+07,1.59575E-09
-a1,structural,9.01479E+07,1.58396E-09
-a1,structural,9.02160E+07,1.57225E-09
-a1,structural,9.02840E+07,1.56064E-09
-a1,structural,9.03521E+07,1.54911E-09
-a1,structural,9.04113E+07,1.53916E-09
-a1,structural,9.04705E+07,1.52927E-09
-a1,structural,9.05297E+07,1.51944E-09
-a1,structural,9.05889E+07,1.50968E-09
-a1,structural,9.06482E+07,1.49999E-09
-a1,structural,9.06998E+07,1.49159E-09
-a1,structural,9.07514E+07,1.48324E-09
-a1,structural,9.08030E+07,1.47494E-09
-a1,structural,9.08547E+07,1.46669E-09
-a1,structural,9.09063E+07,1.45848E-09
-a1,structural,9.13068E+07,1.39645E-09
-a1,structural,9.17072E+07,1.33715E-09
-a1,structural,9.21077E+07,1.28048E-09
-a1,structural,9.25082E+07,1.22629E-09
-a1,structural,9.29086E+07,1.17449E-09
+#,,,,"kind='rlz-000', risk_investigation_time=None"
```

Comment:
If `risk_investigation_time` is not explicitly specified in the job file, the engine uses `risk_investigation_time = investigation_time`

Reply:
Fixed